src/main/java/seedu/address/logic/parser/DateParserUtil.java

```diff
@@ -29,6 +29,8 @@
             + "Please visit our user guide"
             + " for more date/time variants."
             + "\nType help in the input box for the link!";
+    public static final String MESSAGE_EMPTY_DATE = "The Date input should not be empty.\n"
+            + "See the date input format on our User Guide";
```

Comment:
Should we give a sample input here? I think either way would be fine. Let me know what do you think. 👍

Reply:
I don't think we need to give an example here since we are reminding the user to enter a date, it is not that they don't know how to enter the date.

How do the rest think?